news/7402.bugfix

```diff
@@ -0,0 +1 @@
+Raise an exception if revision part of URL is empty for URL used in VCS support
```

Comment:
```suggestion
Abort if a VCS URLs contains an empty revision.
```

Reply:
Thanks @pradyunsg , I have fixed the news message. You can go ahead and merge it :)